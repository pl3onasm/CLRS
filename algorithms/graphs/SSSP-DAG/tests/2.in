r

t z 2 
r t 3
x z 1
r s 5
t x 7
s x 6
y z -2
t y 4
s t 2
x y -1

