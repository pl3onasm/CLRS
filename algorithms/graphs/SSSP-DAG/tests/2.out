
--------------------
 Graph
 Directed
 Weighted
 Vertices: 6
 Edges: 10
--------------------
  r[2]: s(5), t(3)
  s[2]: t(2), x(6)
  t[3]: y(4), x(7), z(2)
  x[2]: y(-1), z(1)
  y[1]: z(-2)
  z[0]
--------------------


Shortest paths
Source: r
---------------------------------
Vertex: Parent, Distance from src
---------------------------------
  r: NIL, 0.00
  z: t, 5.00
  t: r, 3.00
  y: t, 7.00
  s: r, 5.00
  x: t, 10.00
---------------------------------
